lib/WP_Auth0_LoginManager.php

```
@@ -163,136 +170,124 @@ public function init_auth0() {
       } else {
         $this->redirect_login();
       }
-    } catch (WP_Auth0_LoginFlowValidationException $e) {
+    } catch ( WP_Auth0_LoginFlowValidationException $e ) {
 
+      // Errors during the OAuth login flow
       $this->die_on_login( $e->getMessage(), $e->getCode() );
 
-    } catch (WP_Auth0_BeforeLoginException $e) {
+    } catch ( WP_Auth0_BeforeLoginException $e ) {
 
+      // Errors during the WordPress login flow
       $this->die_on_login( $e->getMessage(), $e->getCode(), FALSE );
-
-    } catch (Exception $e) {
```

Empty catch block was catching errors from other plugins